src/app/shared/components/exportCsvFile/export-csv.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExportCSVService {
+  static uploadCsv(blob: Blob, fName: string) {
```

Comment:
Strange that this is named `uploadCsv` but is used in export functions

Reply:
It execute upload a file to local user-PC ... no prob to rename it to exportCSV for example....
Is it acceptable ? (mean exportCsv instead of uploadCsv ?)

Reply:
Rename it to `downloadCsv` and make it a normal function, not `static`

Reply:
look at the functionality - you don't need here an object to serve somehow it, second one - you don't need to inject this functionality, let leave it simple... that's why static where selected. It similar to a React HOOK (functional programming) approach ...  Maybe you will change you opinion about static ?

Reply:
In this case you can remove `@Injectable` annotation if is not used DI

Reply:
Yep - totally agree ...

Reply:
Updated as requested.